keyboards/tanuki/keymaps/tucznak/config.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#undef MANUFACTURER
+#undef PRODUCT
+#undef DESCRIPTION
+
+#define MANUFACTURER    Potato Inc.
+#define PRODUCT         Trash Panda
+#define DESCRIPTION     Qt3.14 smolkeeb
+
+/* for bootloader */
+#define QMK_ESC_OUTPUT B2
+#define QMK_ESC_INPUT D0
+#define QMK_LED B0
+
+/* turn off RGB when computer sleeps */
+#ifdef RGB_DI_PIN
```

Comment:
This is more of a nit-pick, but this would be better: 
```suggestion
#ifdef RGBLIGHT_ENABLE
```